app/test/publisher/api_test.dart

```diff
@@ -61,6 +63,241 @@ void main() {
+            'example.com',
+            InviteMemberRequest(
+                reason: 'Need more admin.', email: testUserA.email));
+        expect(rs.emailSent, isTrue);
+        expect(await queryConstents(testUserA.userId), [
+          {
+            'id': isNotNull,
+            'type': 'PublisherMember',
+            'args': ['example.com'],
+            'notificationCount': 1,
+            'descriptionText': 'be a member of publisher example.com.',
+            'descriptionHtml': 'Be a member of publisher example.com.',
+          }
+        ]);
+      });
+
+      testWithServices('Don not send e-mail twice in a row', () async {
+        final client = createPubApiClient(authToken: hansUser.userId);
+        final rs1 = await client.invitePublisherMember(
+            'example.com',
+            InviteMemberRequest(
+                reason: 'Need more admin.', email: testUserA.email));
+        expect(rs1.emailSent, isTrue);
+        final rs2 = await client.invitePublisherMember(
+            'example.com',
+            InviteMemberRequest(
+                reason: 'Need more admin.', email: testUserA.email));
+        expect(rs2.emailSent, isFalse);
+      });
+
+      testWithServices('Accept invite', () async {
+        final client1 = createPubApiClient(authToken: hansUser.userId);
+        await client1.invitePublisherMember(
+            'example.com',
+            InviteMemberRequest(
+                reason: 'Need more admin.', email: joeUser.email));
+        final consents = await dbService
+            .query<Consent>(ancestorKey: joeUser.key)
+            .run()
+            .toList();
+        final consentId = consents.single.consentId;
+        final client2 = createPubApiClient(authToken: joeUser.userId);
+        final rs2 = await client2.resolveConsent(
+            consentId, account_api.ConsentResult(granted: true));
+        expect(rs2.granted, isTrue);
+        final m =
+            await client1.publisherMemberInfo('example.com', joeUser.userId);
+        expect(m.toJson(), {
+          'userId': 'joe-at-example-dot-com',
+          'isPending': false, // must not be pending!
+          'role': 'admin',
+          'email': 'joe@example.com',
+        });
+      });
+
+      testWithServices('Decline invite', () async {
+        final client1 = createPubApiClient(authToken: hansUser.userId);
+        await client1.invitePublisherMember(
+            'example.com',
+            InviteMemberRequest(
+                reason: 'Need more admin.', email: joeUser.email));
+        final consents = await dbService
+            .query<Consent>(ancestorKey: joeUser.key)
+            .run()
+            .toList();
+        final consentId = consents.single.consentId;
+        final client2 = createPubApiClient(authToken: joeUser.userId);
+        final rs2 = await client2.resolveConsent(
+            consentId, account_api.ConsentResult(granted: false));
+        expect(rs2.granted, isFalse);
```

Comment:
Please test rejecting the invite twice, from where I'm standing that should work, right?